docs/integration/sessions.md

```diff
@@ -0,0 +1,68 @@
+---
+category: integration
+---
+
+# DynamoDB Sessions integration
+
+Store your PHP Sessions in DynamoDB.
+
+## Install
+
+```shell
+composer require async-aws/dynamo-db-session
+```
+
+## Usage
+
+```php
+use AsyncAws\DynamoDb\DynamoDbClient;
+use AsyncAws\DynamoDbSession\SessionHandler;
+
+\session_set_save_handler(new SessionHandler(new DynamoDbClient(), [
+  'table_name' => 'php-sessions',
+]), true);
+```
+
+A DynamoDb table needs to exist in the configured region with the given `table_name` config.
```

Comment:
What's about providing a method to do it?

Reply:
this would require the DynamoDbClient::updateTimeToLive method which doesn't exist yet

Reply:
https://github.com/async-aws/aws/pull/618

Reply:
PR #618 is merged.

Reply:
done!